recipes/terraform-provider-kubectl/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set provider = "kubectl" %}
+{% set goname = "github.com/alekc/terraform-provider-"+provider.lower() %}
+{% set version = "2.0.4" %}
+
+{% set name = goname.split('/')[-1] %}
+
+{% set plugin_dir = os.sep.join(["plugins", "registry.terraform.io", "hashicorp", provider.lower(), version]) %}
+{% set x4_plugin = name + '_v' + version %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+  sha256: 70efbbb75c12d958c489a781f5063e1c1a7fa38e44012489856c554d3278e990
+
+
+build:
+  number: 0
+  script: |
+    go build -v -o {{ target_gobin }}..{{ os.sep }}{{ plugin_dir }}{{ os.sep }}{{ target_goos }}_{{ target_goarch }}{{ os.sep }}{{ x4_plugin }}{{ target_goexe }}
+    # save thirdparty licenses
+    go-licenses save . --save_path ./thirdparty
```

Might possibly need the below:
```suggestion
    go-licenses save . --save_path ./thirdparty --ignore github.com/alekc/terraform-provider-kubectl
```